### ISSUE TITLE
[BUG] status-check endpoint not working as expected

### ISSUE THREAD

**Issue by vpn** [Lissy93/dashy#1416]
### Environment

Self-Hosted (Docker)

### System

Chrome 120

### Version

2.1.1

### Describe the problem

When attempting to query the status-check endpoint of Dashy through Postman, the response received is: "Sorry, JavaScript needs to be enabled to run Dashy". This seems to contradict the expected behavior as outlined in the project's documentation.

As per the [Dashy Documentation](https://github.com/Lissy93/dashy/blob/master/docs/troubleshooting.md#status-checks-failing), this endpoint is recommended for use with Postman to diagnose issues related to failing status checks.

### Additional info

![image](https://github.com/Lissy93/dashy/assets/23225056/1371c6d1-10fc-4906-88e5-2ce76ee896c9)

### Please tick the boxes

- [X] You have explained the issue clearly, and included all relevant info
- [X] You are using a [supported](https://github.com/Lissy93/dashy/blob/master/.github/SECURITY.md#supported-versions) version of Dashy
- [X] You've checked that this [issue hasn't already been raised](https://github.com/Lissy93/dashy/issues?q=is%3Aissue)
- [X] You've checked the [docs](https://github.com/Lissy93/dashy/tree/master/docs#readme) and [troubleshooting](https://github.com/Lissy93/dashy/blob/master/docs/troubleshooting.md#troubleshooting) guide ![important](https://img.shields.io/badge/Important!-F6094E)
- [X] You agree to the [code of conduct](https://github.com/Lissy93/dashy/blob/master/.github/CODE_OF_CONDUCT.md#contributor-covenant-code-of-conduct)

**Comment by bobbyl140** [comment]
I have the same problem with the `status-check` endpoint not working, except I tried loading one of the pages in a browser and got a 404 back from Dashy. Funny enough, two of my 8 services do respond to status checks when loading the Dashy UI. But the `status-check` endpoint for the same URL doesn't load. I even copied the loaded URL from my browser console and it still 404s. I too host myself with Docker.

**Comment by CrazyWolf13** [comment]
Can't get it to work either.

Are you both @bobbyl140  and @vpn using SSL by any chance?

**Comment by bobbyl140** [comment]
I am, have you found that to be an issue?

**Comment by CrazyWolf13** [comment]
> I am, have you found that to be an issue?

Okay so you are using https://[dashy-url].... and you get the issue as well?
Just to rule a relation of ssl (http/https) out.

**Comment by bobbyl140** [comment]
> > I am, have you found that to be an issue?
> 
> Okay so you are using https://[dashy-url].... and you get the issue as well? Just to rule a relation of ssl (http/https) out.

That’s correct, yeah. I have all public domain name pointing to it that I use.

**Comment by bobbyl140** [comment]
Also all the services I tried to status check are HTTPS as well
